app/Controllers/AddressChange/ShowUpdateAddressController.php

```diff
@@ -15,18 +15,17 @@
  */
 class ShowUpdateAddressController {
 
-	public const ADDRESS_CHANGE_SESSION_KEY = 'address_changed';
-
 	public function index( Request $request, FunFunFactory $ffFactory ): Response {
 		$addressToken = $request->get( 'addressToken', '' );
 		if ( $addressToken === '' ) {
-			throw new AccessDeniedException( 'No address change token in request' );
+			throw new AccessDeniedException( 'address_change_no_token_in_request' );
```

Comment:
does the actual translation of that key happens automatically here?

Reply:
Yes, the [error handler passes the message key to the frontend](https://github.com/wmde/fundraising-application/blob/main/app/EventHandlers/HandleExceptions.php#L54)  which [translates the message](https://gitlab.com/fun-tech/fundraising-app-frontend/-/blob/main/src/components/pages/Error.vue)
 I've already added the key to the frontend content repo